src/app/components/LiveHeaderMedia/index.tsx

```diff
@@ -109,7 +109,7 @@ const LiveHeaderMedia = ({
         ]}
         className="hoverStylesText"
       >
-        {showMedia && <VisuallyHiddenText>{closeVideo}, </VisuallyHiddenText>}
+        {showMedia && <VisuallyHiddenText>{translations.media.closeVideo}, </VisuallyHiddenText>}
```

Comment:
We don't need to do this, as we have already destructured `closeVideo` from `translations.media` above (line 52).